Solution1/Solution1.Tests/BL/Extensions/ForecastWeatherDTOExtensionsTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿using BusinessLayer.DTOs;
+using BusinessLayer.Extensions;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Weather.Tests.BL.Extensions
+{
+    public class ForecastWeatherDTOExtensionsTests
+    {
+        [Fact]
+        public void GetMultiStringRepresentation_GetMultiStringRepresentationFromWeatherDTO_Success()
+        {
+            // Arrange
+            var forecastWeatherDTO = GetForecastWeatherDTO().FillCommentByTemp();
```

Comment:
again you call TWO methods in test. you should call only ONE - which you test

Reply:
+